### PR TITLE
Bits from the GitHub Query sandbox app

### PR DESCRIPTION
**Pull request by IEvangelist**
Related to https://github.com/dotnet/docs/issues/36476

### CODE REVIEW

**Comment by gewarren** [comment]
Looks like there are some NRT warnings. Do you plan to address them?

**Comment by IEvangelist** [comment]
Those have been there for as long as I can remember, and they're in a different project altogether. But I can update them.

File: actions/sequester/Quest2GitHub/Models/QuestWorkItem.cs [hunk 1]
```diff
@@ -273,20 +274,25 @@ public static string BuildDescriptionFromIssue(GithubIssue issue, string? reques
             var newItem = QuestWorkItem.WorkItemFromJson(jsonDocument);
             linkedGitHubRepo = true;
             return newItem;
-        } catch (InvalidOperationException ex)
+        }
+        catch (InvalidOperationException ex)
         {
-            Console.WriteLine("Can't add closing PR. The GitHub repo is likely not configured as linked in Quest.");
+            Console.WriteLine($"""
```
Comment (gewarren) [inline actions/sequester/Quest2GitHub/Models/QuestWorkItem.cs:280]: What does the """ do?

**Review by gewarren — APPROVED** [review]

File: actions/sequester/Quest2GitHub/Models/QuestWorkItem.cs (hunk 1 above)
Comment (IEvangelist) [reply actions/sequester/Quest2GitHub/Models/QuestWorkItem.cs:280]: This is the raw string literals: https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/#raw-string-literals

**Pull request merged**